Makefile

```diff
@@ -63,7 +64,10 @@ release-major release-minor release-patch:
 
 .PHONY: setup
 setup:
-	$(NPM) install
+	$(YARN)
```

Comment:
Let's change this to `$(YARN) install --pure-lockfile` to avoid updating __yarn.lock__ here (as that's not our intention).